src/gui/SetupDialog.cpp

```diff
@@ -394,37 +416,19 @@ SetupDialog::SetupDialog( ConfigTabs _tab_to_open ) :
 				embed::getIconPixmap( "project_open", 16, 16 ),
 							"", lmms_wd_tw );
 	workingdir_select_btn->setFixedSize( 24, 24 );
-	workingdir_select_btn->move( 320, 16 );
+	workingdir_select_btn->move( btnStart, 16 );
 	connect( workingdir_select_btn, SIGNAL( clicked() ), this,
 						SLOT( openWorkingDir() ) );
 
-	// vst-dir
-	TabWidget * vst_tw = new TabWidget( tr(
-					"VST-plugin directory" ).toUpper(),
-								paths );
-	vst_tw->setFixedHeight( 48 );
-
-	m_vdLineEdit = new QLineEdit( m_vstDir, vst_tw );
-	m_vdLineEdit->setGeometry( 10, 20, 300, 16 );
-	connect( m_vdLineEdit, SIGNAL( textChanged( const QString & ) ), this,
-					SLOT( setVSTDir( const QString & ) ) );
-
-	QPushButton * vstdir_select_btn = new QPushButton(
-				embed::getIconPixmap( "project_open", 16, 16 ),
-								"", vst_tw );
-	vstdir_select_btn->setFixedSize( 24, 24 );
-	vstdir_select_btn->move( 320, 16 );
-	connect( vstdir_select_btn, SIGNAL( clicked() ), this,
-						SLOT( openVSTDir() ) );
 
 	// artwork-dir
 	TabWidget * artwork_tw = new TabWidget( tr(
-					"Artwork directory" ).toUpper(),
-								paths );
+					"Themes directory" ).toUpper(),
+								pathSelectors );
 	artwork_tw->setFixedHeight( 48 );
 
 	m_adLineEdit = new QLineEdit( m_artworkDir, artwork_tw );
-	m_adLineEdit->setGeometry( 10, 20, 300, 16 );
+	m_adLineEdit->setGeometry( 10, 20, txtLength, 16 );
```

Comment:
> I assume we'll eventually want to do this with all of the repeated variables.

I feel this should be done as a separate pull request. mainly to keep this pull request to functionality. Also to keep the code review more manageable.